src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -128,8 +128,6 @@ def set_tags(scope: Scope, result: JSONData) -> None:
         scope.set_tag(
             "stacktrace_link.auto_derived", result["config"]["automaticallyGenerated"] is True
         )
-    if result.get("codecov") and result["codecov"].get("attemptedUrl"):
```

Comment:
attemptedUrl is only set on error. Instead, let's set the request url and params when we make the request